lib/const.rb

```diff
@@ -13,11 +13,11 @@
 @libcvertokens=  %x[/#{ARCH_LIB}/libc.so.6].lines.first.chomp.split(/[\s]/)
 LIBC_VERSION = @libcvertokens[@libcvertokens.find_index("version") + 1].sub!(/[[:punct:]]?$/,'')
 
-if ENV['CREW_PREFIX'].to_s.empty?
+if ENV['CREW_PREFIX'].to_s.empty? or ENV['CREW_PREFIX'] == '/usr/local'
```

Comment:
I fail to see how this logic solves the problem.  It should still be set in the else block if the environment variable is set.

Reply:
Case 1: CREW_PREFIX is empty
Case 2: CREW_PREFIX = /usr/local
Case 3: CREW_PREFIX is something else

If 1 or 2, set CREW_PREFIX=/usr/local
if 3, leave CREW_PREFIX alone.

Currently Case 2 or 3 breaks `crew`.

Reply:
Shouldn't case 2 and 3 still work with the old logic?

Reply:
Case 2 doesn't work. Try it: `CREW_PREFIX=/usr/local crew const CREW_PREFIX`

Reply:
The only difference with the original logic is it doesn't set `CREW_BUILD_FROM_SOURCE = 1` but I'm thinking better logic would be the following?
```
CREW_PREFIX = '/usr/local'
if ENV['CREW_PREFIX'] and ENV['CREW_PREFIX'] != CREW_PREFIX
  CREW_PREFIX = ENV['CREW_PREFIX']
  CREW_BUILD_FROM_SOURCE = 1
end
```

Reply:
Sounds good to me, and it appears to work too!